public/js/app/models/User.js

```diff
@@ -7,6 +7,14 @@ define(["app/app"], function(App) {
     type: DS.attr('string'),
     screenName: DS.attr('string'),
     email: DS.attr('string'),
-    statistics: DS.attr()
+    statistics: DS.attr(),
+
+    isGroup: function() {
```

Comment:
Never, this is a constant. Why do we need this, you may ask? When we request feed info from URL like /v1/users/epicmonkey we do not know at this stage what is "epicmonkey", i.e. it could be either group or user.